pipelines/deduplication/acteur_deduplicate.py

```diff
@@ -0,0 +1,223 @@
+
+Paramètres optionnelles:
+ - CLUSTER_IDS_TO_CHANGES: pour tester un cluster en particulier
+ - DRY_RUN: pour tester sans modifier la DB
+
+Execution:
+ - python acteur_deduplicate.py
+
+Nommage des variables:
+ - suffixe "_src" pour la donnée de source
+ - suffixe "_db" pour la donnée en base de données
+"""
+
+import json
+import os
+from pathlib import Path
+from typing import List, Tuple
+
+import pandas as pd
+from models.change import Change
+from rich import print
+from rich.prompt import Confirm, Prompt
+from rich.traceback import install
+from sqlalchemy import create_engine
+from sqlalchemy.engine import Engine
+from tasks.db_manage_cluster import db_manage_cluster
+from tasks.source_data_get import source_data_get
+from tests.verifications import CLUSTER_IDS_TO_CHANGES
+from transforms.dataframes import df_mapping_one_to_many
+from utils.cli import banner
+
+install()  # you can install globally to your env, see docs
+
+# ------------------------------------------
+# PARAMETERS
+# ------------------------------------------
+SOURCES_PREFERRED_CODES = ["ALIAPUR", "COREPILE"]
+ENVIRONMENTS = ["DEV", "PREPROD", "PROD"]
+# URLs de la DB
+DB_URLS = {
+    "DEV": os.environ.get("DB_URL_DEV"),
+    "PREPROD": os.environ.get("DB_URL_PREPROD"),
+    "PROD": os.environ.get("DB_URL_PROD"),
+}
+# Chemin vers le fichier CSV
+CURRENT_DIR = Path(__file__).resolve().parent
+CLUSTERING_CSV_FILEPATH = CURRENT_DIR / "./20241121_dedup_dechetteries.csv"
+
+# Colonnes pour le debug
+COLS_DEBUG = ["cluster_id", "identifiant_unique", "children_count", "source_id"]
+
+
+def environment_setup() -> Tuple[Engine, list, Path]:
+    """Choix de l'environnement et retourne les paramètres nécessaires
+
+    Returns:
+        db_engine (Engine): connexion à la base de données
+        clusters_done_list: liste des clusters traités
+        clusters_done_path: chemin vers le fichier de suivi des clusters traités
+    """
+
+    banner("Setup de l'environnement")
+    print("DB_URLS:")
+    for env, url in DB_URLS.items():
+        print(f"{env}={url}")
+    assert any(
+        DB_URLS.values()
+    ), f"""URLs DB manquantes: besoin os.environ sur 1 de
+        {["DB_URL_" + env for env in ENVIRONMENTS]}"""
+
+    env_name = Prompt.ask("Environnement", choices=ENVIRONMENTS)
+    db_url = DB_URLS[env_name]
+    assert db_url, "db_url est vide"
+    db_engine = create_engine(db_url)
+
+    # Fichier de suivi des clusters traités construit à partir de l'environnement
+    clusters_done_path = CURRENT_DIR / f"./clusters_done_{env_name.lower()}.json"
+    clusters_done_list = []
+    if not clusters_done_path.exists():
+        clusters_done_path.write_text(json.dumps(clusters_done_list, indent=4))
+    else:
+        clusters_done_list = json.loads(clusters_done_path.read_text())
+
+    print(f"{env_name=}")
+    print(f"{db_url=}")
+    print(f"{db_engine=}")
+    print(f"{clusters_done_path=}")
+    print(f"{len(clusters_done_list)=}")
```

Comment:
```suggestion
    print(f"{env_name=}")
    print(f"{db_url=}")
    print(f"{db_engine=}")
    print(f"{clusters_done_path=}")
    print(f"{len(clusters_done_list)=}")
```

Veut-on vraiment print la db_url en clair dans les logs ?